README.md

```diff
@@ -93,12 +93,23 @@ create /etc/eremetic/eremetic.yml with:
     logformat: json
 
 ## Database Backing
-Eremetic uses a BoltDB based database to store task information. The location of
-it can be set by adding
+Eremetic uses a database to store task information. The driver can be configured
+by setting the `database_driver` value to either `zk` or `boltdb`.
 
-    database: '/tmp/eremeticdb/eremetic.db'
+The location of the database can be configured by setting the `database` value.
 
-to the eremetic.yml
+### BoltDB
+The default database that will be used unless anything is configured.
+
+The default location for it is `db/eremetic.db`
+
+### ZooKeeper
+If you use `zk` as a database driver, the default location of the database is set
+to the zookeeper configured in the `master` field, but `/mesos` will be replaced
+by `/eremetic`.
+
+Note:
+`/eremetic` will always be appended to a `zk` database path.
```

Comment:
Yes, I agree with that.

I guess not having a default would be the best solution, and just taking the path as it is being sent it

Problem is the zk client doesn't seem to accept a zk:// string as it's argument, so would need to split it up somehow, but I'll figure something out.